src/Localization/Abstractions/ref/Microsoft.Extensions.Localization.Abstractions.csproj

```diff
@@ -3,6 +3,7 @@
   <PropertyGroup>
     <TargetFrameworks>netstandard2.0;$(DefaultNetCoreTargetFramework)</TargetFrameworks>
     <TargetFrameworks Condition="'$(DotNetBuildFromSource)' == 'true'">$(DefaultNetCoreTargetFramework)</TargetFrameworks>
+    <Nullable>annotations</Nullable>
```

Comment:
Ick, this gets ugly duplicating the src/ project's properties. Is it possible to add the (conditional) property in eng/targets/CSharp.ReferenceAssembly.props. I'm betting that would require a target in the src/ project that returns `$(Nullable)` and that's a _major_ hack as well as an `<MSBuild/>` explosion. I hope there's something we could do but don't have a good suggestion.

Another bad suggestion 😈 would be to load the src/ project into a `string` and see if it contains `<Nullable>annotations</Nullable>` or `<Nullable>enable</Nullable>`

Reply:
Using `<Nullable>enable</Nullable>` in the ref assembly causes build errors because of all the of `throw null;` for instance, from the ref assembly below:

```C#
public string Value { [System.Runtime.CompilerServices.CompilerGeneratedAttribute] get { throw null; } }
```


Always using `annotations` seems like the right way unless we change the codegen to be null-safe.